elements/lisk-p2p/test/config/jest.config.base.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+module.exports = {
+	testEnvironment: 'node',
+	verbose: true,
+	collectCoverage: true,
+	coverageReporters: ['json', 'lcov', 'cobertura'],
+	rootDir: '../../../../',
```

Comment:
Why root dir is here? It should be independent of root path

Reply:
It is needed to load jest-extend from the hoisted node_modules.